packages/api/src/platforms/vtex/clients/commerce/index.ts

```diff
@@ -108,6 +108,53 @@ export const VtexCommerce = (
           : Promise.resolve(undefined);
       },
 
+      getDeliveryWindows: ({
+        id,
+        index,
+        deliveryMode,
+        body,
+      }: {
+        id: string
+        index: number
+        deliveryMode?: DeliveryMode | null
+        body: ShippingDataBody
+      },
+        incrementedAddress?: IncrementedAddress): Promise<OrderForm> => {
+
+        const mappedBody = {
+          "logisticsInfo": Array.from({ length: index }, (_, itemIndex) => ({
+            itemIndex,
+            selectedDeliveryChannel: deliveryMode?.deliveryChannel,
+            selectedSla: deliveryMode?.deliveryMethod,
+          })),
+          "selectedAddresses": body?.selectedAddresses?.map(address => ({
+            "addressType": address.addressType || null,
+            "receiverName": address.receiverName || null,
+            "postalCode": address.postalCode || incrementedAddress?.postalCode || null,
+            "city": incrementedAddress?.city || null,
+            "state": incrementedAddress?.state || null,
+            "country": address.country || incrementedAddress?.country || null,
+            "street": incrementedAddress?.street || null,
+            "number": incrementedAddress?.number || null,
+            "neighborhood": incrementedAddress?.neighborhood || null,
+            "complement": incrementedAddress?.complement || null,
+            "reference": incrementedAddress?.reference || null,
+            "geoCoordinates": address.geoCoordinates || incrementedAddress?.geoCoordinates || []
+          }))
+        };
```

Comment:
You need run the prettier here

Reply:
Done! I was without prettier. Sorry about that!